models/filter/paymail_filter.go

```diff
@@ -0,0 +1,34 @@
+package filter
+
+// PaymailFilter is a struct for handling request parameters for paymail_addresses search requests
+type PaymailFilter struct {
+	ModelFilter `json:",inline"`
+
+	ID                 *string `json:"id,omitempty"`
+	Alias              *string `json:"alias,omitempty"`
+	Domain             *string `json:"domain,omitempty"`
+	PublicName         *string `json:"publicName,omitempty"`
+	Avatar             *string `json:"avatar,omitempty"`
```

Comment:
Maybe someone who has put there some data... and since it is a string I decided to keep it. However, maybe, it's better to remove it for now and eventually add when such requirement appears.

Reply:
I would do as you said: remove now, and if someone need it, then we will add

Reply:
Removed